dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -409,20 +409,19 @@ export const Card = ({
 							displayAge={displayAge}
 							renderFooter={renderFooter}
 						/>
-
-						{hasSublinks && noOfSublinks <= 2 ? (
-							<SupportingContent
-								supportingContent={supportingContent}
-								alignment="vertical"
-								containerPalette={containerPalette}
-								isDynamo={isDynamo}
-							/>
-						) : (
-							<></>
-						)}
```

Comment:
It seems there are situations where Frontend puts the sublinks 'alongside' the image:

Frontend:
<img width="1336" alt="image" src="https://user-images.githubusercontent.com/9575458/200852549-8f16d4f6-d47e-4be8-9462-ee3bc7c3052d.png">

DCR (main)
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/9575458/200852712-b187baf2-2d13-485c-bd3e-ead11a9239fb.png">


DCR (this branch)
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/9575458/200852604-cdd784b3-5f61-4a6f-b7ae-1837aeb7caa0.png">

I think we might want to re-visit the logic that decides whether Supporting content should be alongside the image, or below it!